### PR TITLE
added tvsquared as destination

### PR DESCRIPTION
**Pull request by ruchiramoitra**
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/120)

### CODE REVIEW

File: integrations/TVSquared/browser.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    window._tvq.push([
+      function () {
+        this.deleteCustomVariable(5, "page");
+      },
+    ]);
+  }
+
+  isLoaded() {
+    logger.debug("in TVSqaured isLoaded");
+    return !!window_tvq;
```
Comment (prabrishac) [inline integrations/TVSquared/browser.js:29]: typo? should be !!window._tvq

should we not do something like - 
return !!(window._tvq && window._tvq.push !== Array.prototype.push);

File: integrations/TVSquared/browser.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  }
+
+  isReady() {
+    logger.debug("in TVSqaured isReady");
+
+    return !!window_tvq;
```
Comment (prabrishac) [inline integrations/TVSquared/browser.js:35]: same as  `isLoaded`

File: integrations/TVSquared/browser.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+          action[key] = value;
+        }
+      }
+    }
+    window._tvq.push([
+      function () {
+        this.setCustomVariable(5, "session", JSON.stringify(session), "visit");
+      },
+    ]);
+    if (event !== "Response") {
```
Comment (prabrishac) [inline integrations/TVSquared/browser.js:89]: or `All response`
Should we make the check case insensitive?

File: integrations/integration_cname.js [hunk 4]
```diff
@@ -42,6 +42,7 @@ const commonNames = {
   FULLSTORY: "FULLSTORY",
   Fullstory: "FULLSTORY",
   BUGSNAG: "BUGSNAG",
+  TVSQUARED: "TVSQUARED",
```
Comment (prabrishac) [inline integrations/integration_cname.js:45]: also put corresponding entry in `client_server_name.js`

File: integrations/ScriptLoader.js [hunk 5]
```diff
@@ -6,9 +6,13 @@ const ScriptLoader = (id, src) => {
   js.src = src;
   js.async = true;
   js.type = "text/javascript";
+  if(id === "TVSquared-integration"){
+    js.defer = true;
+  }
```
Comment (prabrishac) [inline integrations/ScriptLoader.js:11]: currently we load all js as `async` only. remove the `defer` attribute now. we have the task of making all the scripts with `defer` in pipeline. Will do the change as part of that task.

File: integrations/ScriptLoader.js [hunk 6]
```diff
@@ -6,9 +6,13 @@ const ScriptLoader = (id, src) => {
...
   js.id = id;
   const e = document.getElementsByTagName("script")[0];
   logger.debug("==script==", e);
+  console.log("==script==", e)
```
Comment (prabrishac) [inline integrations/ScriptLoader.js:15]: remove all `console.log`

File: integrations/ScriptLoader.js (hunk 5 above)
Comment (ruchiramoitra) [reply integrations/ScriptLoader.js:11]: ok

File: integrations/TVSquared/browser.js (hunk 1 above)
Comment (ruchiramoitra) [reply integrations/TVSquared/browser.js:29]: yes it was a type changed it.

File: integrations/TVSquared/browser.js (hunk 3 above)
Comment (ruchiramoitra) [reply integrations/TVSquared/browser.js:89]: All response was not mentioned in the sdk doc so did not include it. Doing the case insensitive part

File: integrations/integration_cname.js (hunk 4 above)
Comment (ruchiramoitra) [reply integrations/integration_cname.js:45]: integrations/client_server_name.js

I have put here..

File: integrations/TVSquared/browser.js (hunk 2 above)
Comment (ruchiramoitra) [reply integrations/TVSquared/browser.js:35]: done

File: integrations/integration_cname.js (hunk 4 above)
Comment (prabrishac) [reply integrations/integration_cname.js:45]: ohh... missed it.

**Review by arnab-p — APPROVED** [review]

**Pull request merged**